frb_example/pure_dart/dart/lib/bridge_generated.dart

```diff
@@ -540,6 +564,14 @@ class Point {
   });
 }
 
+class Sequences {
+  final Int32List field0;
+
+  Sequences({
+    required this.field0,
+  });
+}
```

Comment:
So is this expected behavior? Numbers and Sequences are still two classes, not one

Reply:
@fzyzcjy Yes, this is the desired result, less template code and more mirrors generated.